freeform_json_to_yaml.py

```diff
@@ -0,0 +1,440 @@
+                        'status': 'hidden'
+                    }
+                    written_injuries[injury['name']] = ['internal']
+                elif i['type'] == 'Ear Bleed':
+                    injury = {
+                        'name': 'Ear Bleed',
+                        'location': "right face" if randint(1, 2) > 1 else "left face",
+                        'status': 'discoverable'
+                    }
+                    written_injuries[injury['name']] = [injury['location']]
+                else:
+                    if i['type'] == 'Forehead Scrape':
+                        injury = {
+                            'name': "Laceration",
+                            'location': "left face",
+                            'status': 'discoverable'
+                        } 
+                    else:
+                        split_name = i['type'].split(' ')
+                        injury = {
+                            'name': split_name[-1] if 'Broken' not in i['type'] else 'Broken Bone',
+                            'location': INJ_LOC_MAP[(' ').join(split_name[:-1])],
+                            'status': 'discoverable'
+                        }
+                    bloodPool = i.get('bloodPool', None)
+                    if bloodPool is not None:
+                        severity = SEVERITY_MAP[bloodPool]
+                    elif 'Burn' in i['type']:
+                        severity = 'major'
+
+                    injury['severity'] = severity
+                    
+                    if injury['name'] in written_injuries:
+                        written_injuries[injury['name']].append(injury['location'])
+                    else:
+                        written_injuries[injury['name']] = [injury['location']]
+                injuries.append(injury)
+            new_c['injuries'] = injuries
+            if len(written_injuries) == 0:
+                unstructured_inj = 'Has no known injuries.'
+            else:
+                unstructured_inj = "Has"
+                pronoun = 'his' if demographics['sex'] == 'M' else 'her'
+                is_asthmatic = False
+                for inj_set in written_injuries:
+                    if len(written_injuries[inj_set]) == 1:
+                        if inj_set == 'Collapse':
+                            unstructured_inj += f" a collapsed {written_injuries[inj_set][0]},".lower()
+                        elif inj_set == "Shrapnel":
+                            unstructured_inj += f" shrapnel in {pronoun} {written_injuries[inj_set][0]},".lower()
+                        elif inj_set == "Amputation":
+                            unstructured_inj += f" an amputated {written_injuries[inj_set][0]},".lower()
+                        elif inj_set == "Asthmatic":
+                            is_asthmatic = True
+                        elif inj_set == 'Ear Bleed':
+                            unstructured_inj += f" an ear bleed,"
```

Comment:
How about `" blood coming out of an ear,"` ?
I don't know-- is an "earbleed" like a "nosebleed"?  I guess so?
I guess you can change it to "earbleed" instead.  I'm just not sure I've heard that term, but I guess it's a thing...